code/go/0chain.net/smartcontract/partitions/partitions.go

```diff
@@ -55,6 +55,16 @@ func GetPartitions(state state.StateContextI, name string) (*Partitions, error)
 	return &Partitions{rs: &rs}, nil
 }
 
+// GetCommonContextPartitions returns partitions of given name for test handler
+func GetCommonContextPartitions(state state.CommonStateContextI, name string) (*Partitions, error) {
```

Comment:
We change change the state to CommonStateContextI for GetPartitions() instead of add a new function.